datasets/timit_asr/README.md

```diff
@@ -224,4 +224,4 @@ LDC User Agreement for Non-Members
 
 ### Contributions
 
-Thanks to [@patrickvonplaten](https://github.com/patrickvonplaten) for adding this dataset.
\ No newline at end of file
+Thanks to [@patrickvonplaten](https://github.com/patrickvonplaten) for adding this dataset.
```

Comment:
I didn't really do anything to add the dataset - it was all you, so absolutely no need to add me here. You can add yourself if you want ;-)